src/functional/java/reposense/Entry.java

```diff
@@ -28,25 +28,23 @@
     private static final String FT_TEMP_DIR = "ft_temp";
     private static final String EXPECTED_FOLDER = "expected";
 
-    @BeforeClass
-    public static void setUp() throws IOException {
+    @Before
```

Comment:
rationale for changing this?

Reply:
>rationale for changing this?

As the report is now generated in the given output folder instead of having an extra timestamp folder, and since both tests specify the same output directory `ft_temp`, the folder from the previous test needs to be removed 1st in order before re-generating the report for the next test.